src/components/AdvancedSearch.vue

```diff
@@ -40,13 +42,15 @@
     <div class="search-footer">
       <span id="example" v-if="$route.path === '/'">
         <b-button
+          id="exampleButton"
           @click="exampleSearch"
           title="Insert example search parameters to the form"
           >Example range query</b-button
         ></span
       >
       <span id="basicSearch"
         ><b-button
+          id="basic"
```

Comment:
if you added these just for test you could have used: `data-testid` attribute to assign a value just for test

Reply:
I'll change those

Reply:
The tests break if I change the id

Reply:
I think you need to do `const button = document.querySelector("[data-testid=id]");`